optimum/bettertransformer/transformation.py

```diff
@@ -51,8 +51,9 @@ def replace_to_bettertransformer(model, config):
     for name, module in model.named_children():
         if len(list(module.children())) > 0:
             # we may explicitly exclude part of the model to use BetterTransformer
-            if config.model_type not in EXCLUDE_FROM_TRANSFORM or (
-                config.model_type in EXCLUDE_FROM_TRANSFORM and name not in EXCLUDE_FROM_TRANSFORM[config.model_type]
+            if config.model_type not in BetterTransformerManager.EXCLUDE_FROM_TRANSFORM or (
```

Comment:
Since this check is made multiple times, what about exposing methods in the `BetterTransformerManager` to check that. 

Like a method called `is_excluded(self, model_type: str) -> bool`.